crypto/compactcert/builder_test.go

```diff
@@ -69,45 +71,48 @@ func TestBuildVerify(t *testing.T) {
 	npart := npartHi + npartLo
 
 	param := Params{
-		Msg:          TestMessage("hello world"),
-		ProvenWeight: uint64(totalWeight / 2),
-		SigRound:     0,
-		SecKQ:        128,
+		Msg:               TestMessage("hello world"),
+		ProvenWeight:      uint64(totalWeight / 2),
+		SigRound:          currentRound,
+		SecKQ:             128,
+		CompactCertRounds: 128,
 	}
 
 	// Share the key; we allow the same vote key to appear in multiple accounts..
-	key := crypto.GenerateOneTimeSignatureSecrets(0, 1)
+	key, err := merklekeystore.New(0, uint64(param.CompactCertRounds)+1, param.CompactCertRounds, crypto.DilithiumType)
+
+	require.NoError(t, err, "failed to create keys")
 
 	var parts []Participant
-	var sigs []crypto.OneTimeSignature
+	var sigs []merklekeystore.Signature
```

Comment:
please add a comment clarifying that the test creates participants with big weight and little weight.